src/db/import/ClimbTransformer.ts

```diff
@@ -28,7 +28,8 @@ const transformClimbRecord = (row: any): ClimbType => {
       ewbank: grade.Ewbank,
       font: grade.Font,
       french: grade.French,
-      uiaa: grade.UIAA
+      uiaa: grade.UIAA,
+      brazilian_crux: grade.BrazilianCrux
```

Comment:
Here I'm also not sure if `BrazilianCrux` is the right prop.

Reply:
looks right to me - that's the name you gave the scale

Reply:
This is something that the tests really should be able to verify, but we don't have the infrastructure to iterate over all extant scales in sandbag.  The broken features of https://github.com/OpenBeta/sandbag/issues/160 would do much of that if fixed...